docs/book/src/getting-started.md

```diff
@@ -90,9 +90,98 @@ The capability to set credentials using environment variables is now deprecated
 
 
 ### Building your first cluster
-Check out the [Cluster API Quick Start](https://cluster-api.sigs.k8s.io/user/quick-start.html) to create your first Kubernetes cluster on Azure using Cluster API. Make sure to select the "Azure" tabs.
 
-If you are looking to install additional ASO CRDs, set `ADDITIONAL_ASO_CRDS` to the list of CRDs you want to install. Refer to adding additional CRDs for Azure Service Operator [here](./topics/aso.md#Using-aso-for-non-capz-resources).
+The recommended way to build a cluster is to install a CAPZ management cluster using [Cluster API Operator](https://github.com/kubernetes-sigs/cluster-api-operator), then utilizing a helm chart to create a workload cluster, specifically an [ASO Managed Cluster](./managed/asomanagedcluster.md).
+
+To create a cluster manually, check out the [Cluster API Quick Start](https://cluster-api.sigs.k8s.io/user/quick-start.html) for in-depth instructions. Make sure to select the "Azure" tabs. If you are looking to install additional ASO CRDs, set `ADDITIONAL_ASO_CRDS` to the list of CRDs you want to install. Refer to adding additional CRDs for Azure Service Operator [here](./topics/aso.md#Using-aso-for-non-capz-resources).
+
+## Creating a CAPZ Management Cluster with CAPI Operator
+
+First, it is a common practice to create a temporary, local bootstrap cluster to deploy the management cluster. You can either use an existing Kubernetes cluster, or create a kind cluster for this purpose.
+
+```bash
+kind create cluster
+```
+
+Add the CAPI Operator & cert manager helm repository, which we will use to install the management cluster.
+
+```bash
+helm repo add capi-operator https://kubernetes-sigs.github.io/cluster-api-operator
+helm repo add jetstack https://charts.jetstack.io --force-update
+helm repo update
+```
+
+Install cert manager:
+
+```bash
+helm install cert-manager jetstack/cert-manager --namespace cert-manager --create-namespace --set installCRDs=true
+```
+
+Create a `values.yaml` file for the CAPI Operator helm chart like so:
+
+```yaml
+core: "cluster-api:v1.8.5"
+infrastructure: "azure:v1.17.2"
+addon: "helm:v0.2.5"
+manager:
+  featureGates:
+    core:
+      ClusterTopology: true
+```
+
+Install the CAPI Operator:
+
+```bash
+helm install capi-operator capi-operator/cluster-api-operator --create-namespace -f values.yaml --wait --timeout 90s
+```
+
+## Creating an ASO Managed workload cluster
+
+Once your management cluster is up and running, you can create an ASO Managed Cluster using the [azure-aks-aso helm chart](https://github.com/mboersma/cluster-api-charts/tree/main/charts/azure-aks-aso).
+
+Add the cluster-api-charts helm repository:
+
+```bash
+helm repo add capi https://mboersma.github.io/cluster-api-charts
+```
+
+Specify values for the CAPZ AKS-ASO helm chart in a `values.yaml` file. For exmaple:
+
+```yaml
```

Comment:
I'd say to specify here ONLY the minimum values required to provision a cluster and use the default values file provided with the chart.